misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepScanScreen.jsx

```diff
@@ -24,10 +24,11 @@ function PickStepScanScreen(WrappedComponent) {
       activityId,
       lineId,
       stepId,
+      altStepId,
       stepProps,
       appId,
-      qtyTarget: stepProps.qtyToPick,
-      eligibleBarcode: stepProps.huBarcode,
+      qtyTarget: altStepId ? stepProps.altSteps.genSteps[altStepId].qtyAvailable : stepProps.qtyToPick,
```

Comment:
minor: (for alt step) shouldn't we call this `qtyToPick` instead of `qtyAvailable`?